test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -70,6 +75,21 @@ private Layout get3NodeLayout() throws Exception {
                 UUID.randomUUID());
     }
 
+    private Layout get1NodeLayout() {
```

Comment:
Can you add this in the harness (think it already has such utility). Too much boilerplate code.